src/utils/store-utils.ts

```diff
@@ -0,0 +1,9 @@
+import { devtools } from 'zustand/middleware';
+
+// TODO - add in Immer middleware
+// https://github.com/pmndrs/zustand#middleware
+// https://github.com/pmndrs/zustand/blob/main/tests/middlewareTypes.test.tsx
+
+export const devtoolsInNonProd = (process.env.NODE_ENV === 'production'
+    ? (fn: any) => fn
+    : devtools) as unknown as typeof devtools;
```

Comment:
This is a quick and dirty wrapper for including devtools only for non prod. Eventually we should add some common global middleware and maybe even wrap the create function here. But that will take some effort from a typing standpoint.